### PR TITLE
Point Solana feeds at transmissionsAccount

### PR DESCRIPTION
**Pull request by dwightjl**

### CODE REVIEW

File: src/features/feeds/components/Tables.tsx [hunk 1]
```diff
@@ -85,15 +85,15 @@ const DefaultTr = ({ network, proxy, showExtraDetails, isTestnet = false }) => (
...
         >
           <img src="/assets/icons/copyIcon.svg" alt="copy to clipboard" />
         </button>
         <a
           class={tableStyles.addressLink}
-          href={network.explorerUrl.replace("%s", proxy.proxyAddress ?? proxy.contractAddress)}
+          href={network.explorerUrl.replace("%s", proxy.proxyAddress ?? proxy.transmissionsAccount)}
         >
-          {proxy.proxyAddress ?? proxy.contractAddress}
+          {proxy.proxyAddress ?? proxy.transmissionsAccount}
```
Comment (aalu1418) [inline src/features/feeds/components/Tables.tsx:96]: might be nice to add a comment pointing out that pointing to the transmissionsAccount is solana specific as solana does not have a proxyAddress

**Review by aalu1418 — APPROVED** [review]
looks good to me!

**Review by thedriftofwords — CHANGES_REQUESTED** [review]
If you have a moment to address this comment: https://github.com/smartcontractkit/documentation/pull/1118/files#r1080642944

File: src/features/feeds/components/Tables.tsx (hunk 1 above)
Comment (dwightjl) [reply src/features/feeds/components/Tables.tsx]: Added comments

**Comment by dwightjl** [comment]
> If you have a moment to address this comment: https://github.com/smartcontractkit/documentation/pull/1118/files#r1080642944

Done

**Review by thedriftofwords — APPROVED** [review]

**Pull request merged**